k2/csrc/nvtx.cu

```diff
@@ -0,0 +1,25 @@
+/**
+ * @brief
+ * nvtx.cu
+ *
+ * @copyright
+ * Copyright (c)  2020  Mobvoi Inc.        (authors: Fangjun Kuang)
+ *
+ * @copyright
+ * See LICENSE for clarification regarding multiple authors
+ */
+
+#include "k2/csrc/nvtx.h"
+#include "nvToolsExt.h"
+
+namespace k2 {
+
+#ifdef K2_ENABLE_NVTX
+NvtxRange::NvtxRange(const char *name) { nvtxRangePushA(name); }
```

Comment:
I'd have put these in the header, since they are so lightweight-- no reason having an extra function call in the mix.

Reply:
Fixed.